Druid/DRCogitStackToRegisterMappingGenerator.class.st

```diff
@@ -12,24 +12,44 @@ Class {
 DRCogitStackToRegisterMappingGenerator >> allocateVariable: aVariableName [
```

Comment:
What is `aVariableName`? I think it is `aDRResult`. In that case, we could rename the variable... and the method.